### ISSUE TITLE
[Darkmoon Fair] Professor Thaddeus Paleo is selling TBC items

### ISSUE THREAD

**Issue by Hacki95** [azerothcore/mod-progression-system#82]
The Darkmoon Fair NPC "Professor Thaddeus Paleo" is still selling TBC potions.

We need queries to remove those and re-add them once TBC starts.

**Comment by Eronox** [comment]
## WoTLK - 3.0.2 potions

**DELETE**
```
DELETE FROM `npc_vendor` WHERE (`entry` = 14847) AND (`item` IN (33457, 33458, 33459, 33460, 33461, 33462, 37091, 37093, 37097, 43463, 43465, 43467, 33447));
```

**INSERT**
```
INSERT INTO `npc_vendor` (`entry`, `slot`, `item`, `maxcount`, `incrtime`, `ExtendedCost`, `VerifiedBuild`) VALUES
        (14847, 0, 33457, 2, 120, 0, 0),
	(14847, 0, 33458, 2, 120, 0, 0),
	(14847, 0, 33459, 2, 120, 0, 0),
	(14847, 0, 33460, 2, 120, 0, 0),
	
	(14847, 0, 33461, 2, 120, 0, 0),
	(14847, 0, 33462, 2, 120, 0, 0),
	(14847, 0, 37091, 2, 120, 0, 0),
	(14847, 0, 37093, 2, 120, 0, 0),
	
	(14847, 0, 37097, 2, 120, 0, 0),
	(14847, 0, 43463, 2, 120, 0, 0),
	(14847, 0, 43465, 2, 120, 0, 0),
	(14847, 0, 43467, 2, 120, 0, 0),
	(14847, 0, 33447, 2, 120, 0, 0);
```

## TBC - 2.5.1 potions

**DELETE**
```
DELETE FROM `npc_vendor` WHERE (`entry` = 14847) AND (`item` IN (22829, 22832, 27498, 27499, 27500, 27501, 27502, 27503));
```
**INSERT**
```
INSERT INTO `npc_vendor` (`entry`, `slot`, `item`, `maxcount`, `incrtime`, `ExtendedCost`, `VerifiedBuild`) VALUES
	(14847, 0, 22829, 2, 18000, 0, 0),
	(14847, 0, 22832, 2, 18000, 0, 0),
	(14847, 0, 27498, 2, 7200, 0, 0),
	(14847, 0, 27499, 2, 7200, 0, 0),
	(14847, 0, 27500, 1, 7200, 0, 0),
	(14847, 0, 27501, 1, 7200, 0, 0),
	(14847, 0, 27502, 2, 7200, 0, 0),
	(14847, 0, 27503, 2, 7200, 0, 0);
```

**Comment by Eronox** [comment]
This is how I produced above list:

1. Used Keira to see list of all loot
2. Cross-referenced to tbc.wowhead.com using the entry id, looking at `Added in patch` column
3. Filled in the ID in delete
4. After done both deletes, I made a INSERT and copied from `npc_vendor.sql` using CTRL+F on each ID to make sure the origin values comes back


Despite this, it's probably a good idea to test it. 
Let me know if you want a PR of this, as I'm unsure how this should be delivered properly 😄
